core-rust/state-manager/src/transaction/preview.rs

```diff
@@ -150,7 +150,9 @@ mod tests {
             &logging_config,
             FeeReserveConfig::default(),
         ));
-        let pending_transaction_result_cache = Arc::new(parking_lot::const_rwlock(
+        let pending_transaction_result_cache: Arc<
+            parking_lot::lock_api::RwLock<parking_lot::RawRwLock, PendingTransactionResultCache>,
+        > = Arc::new(parking_lot::const_rwlock(
```

Comment:
unintentional change?

Reply:
Totally unintentional, `spotless` decided it should do it.